allensdk/brain_observatory/vbn_2022/metadata_writer/metadata_writer.py

```diff
@@ -197,6 +204,30 @@ def run(self):
                     session_id_col='ecephys_session_id',
                     on_missing_file=self.args['on_missing_file'])
 
+        ecephys_session_ids = ecephys_session_table['ecephys_session_id']
+        behavior_ecephys_session_ids = behavior_session_table[
+            'ecephys_session_id']
+        ecephys_session_mask = behavior_ecephys_session_ids.isin(
+            ecephys_session_ids)
+
+        behavior_only_table = behavior_session_table[
+            ~ecephys_session_mask]
+        behavior_w_ecephy_table = behavior_session_table[
+            ecephys_session_mask]
+        # Fill in null values for file ID to preserve typing of the id
+        # column.
+        behavior_w_ecephy_table['file_id'] = file_id_generator.dummy_value
+        behavior_only_table = add_file_paths_to_metadata_table(
```

Comment:
Please add a comment that we are only adding a file path to the behavior sessions without a corresponding ecephys session or reduce the number of variables used. The logic gets jumbled up in the previous lines making it hard to read.

Reply:
How does this read? Added just above line 220.

"Compute a file_id and add the file path to the table for behavior sessions with no corresponding ecephys session. For behavior sessions with ecephys data, we leave the file_id and file_path as Null values, loading their behavior data from the ecephys session."

Reply:
Reads good to me